src/graph_compute/bsp_scheduling.h

```diff
@@ -45,6 +46,6 @@ static inline void print_execution_data() {
   // no-op
 }
 
-#endif  // D0_BSP
+#endif  // D0_BSP || BASELINE
```

Comment:
This looks like it was left over, but BSP is not BASELINE like we determined, and this doesn't match the `#if` at the top.